### PR TITLE
Add docs for UniFi Protect number platform

### PR DESCRIPTION
**Pull request by AngellusMortis**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [X] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: https://github.com/home-assistant/core/pull/63220

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/unifiprotect.markdown [hunk 1]
```diff
@@ -94,6 +96,11 @@ Each UniFi Protect camera will get the following entities added:
 
 * **Camera** - A camera entity for each camera channel and RTSP(S) combination found for each camera (up to 6). Only the highest resolution RTSPS camera entity will be enabled by default.
 * **Media Player** - If your camera has a speaker, you will get a media player entity that allows you to play audio to your camera's speaker. Any audio file URI that is playable by FFmpeg will be able to be played to your speaker, including via the [TTS Say Service](https://www.home-assistant.io/integrations/tts/#service-say).
+* **Number** - The following number entities will be added for each camera:
```
Comment (frenck) [inline source/_integrations/unifiprotect.markdown:99]: I really do think the documentation structure doesn't make sense.

People look at the information they can have access to and don't look them up by the platform.

File: source/_integrations/unifiprotect.markdown [hunk 2]
```diff
@@ -110,7 +117,10 @@ Each UniFi Protect camera will get the following entities added:
 Each UniFi Protect floodlight will get the following entities added:
 
 * **Light** - A light entity will be added for each floodlight device. The light entity will let you control turning on or off your light as well as adjust the brightness of your floodlight.
-* **Switch** - The following switch entities will be added for each camera:
+* **Number** - The following number entities will be added for each floodlight:
```
Comment (frenck) [inline source/_integrations/unifiprotect.markdown:120]: Same as above

File: source/_integrations/unifiprotect.markdown (hunk 1 above)
Comment (AngellusMortis) [reply source/_integrations/unifiprotect.markdown:99]: Okay.... do you have a suggestion on how to make it better?

Comment (frenck) [reply source/_integrations/unifiprotect.markdown:99]: The problem is that the main chapter has devices/types that also are in this list here as platforms.

Now you end up with sensors that provide numbers and sensors that have cameras, and cameras having media players... the confusion here is that an end-user will have no clue in context between the difference of an actual device vs an entity platform.

I think it is too complex, to begin with. For example: `#### Entities`
While Home Assistant indeed has entities, and those drive the system, from a user perspective we work with devices & services. Entities are part of that system.

I think it makes more sense to talk in term of functionality.

For example; this integration supports UniFi protect cameras (and just that).
You could extend it with a more functional description of what can be controlled.

Comment (AngellusMortis) [reply source/_integrations/unifiprotect.markdown:99]: I think I get what you mean. Should I refactor this PR for these changes (and the other ones you mentioned in previous reviews)? Or should I let this one go as is and do it all in a follow up?

Comment (frenck) [reply source/_integrations/unifiprotect.markdown:99]: Let do a refactor afterward 👍

**Review by frenck — APPROVED** [review]

**Pull request merged**